Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs

```diff
@@ -45,10 +45,10 @@ public class AccessibilityCheckingConfiguration
     /// Gets or sets a predicate that determines whether accessibility checking and asserting the results should run for
     /// the current page. This is only used if <see cref="RunAccessibilityCheckingAssertionOnAllPageChanges"/> is set to
     /// <see langword="true"/>. Defaults to <see
-    /// cref="EnableOnValidatablePagesAccessbilityCheckingAndAssertionOnPageChangeRule"/>.
+    /// cref="EnableOnValidatablePagesAccessibilityCheckingAndAssertionOnPageChangeRule"/>.
     /// </summary>
-    public Predicate<UITestContext> AccessbilityCheckingAndAssertionOnPageChangeRule { get; set; } =
-        EnableOnValidatablePagesAccessbilityCheckingAndAssertionOnPageChangeRule;
+    public Predicate<UITestContext> AccessibilityCheckingAndAssertionOnPageChangeRule { get; set; } =
```

Comment:
This is a breaking change, so re-add the property with the old name with `[Obsolete]`, and let it just proxy to this one.

Reply:
But that won't work, since nothing uses it.

Reply:
This, i.e. a proxy property, is what I've recommended.